steamclog/consumer-rules.pro

```diff
@@ -1,8 +1,8 @@
 # Required if you want to use Steamclog Redactable classes, and Proguard/R8 to obsfucate code.
 -keep class * extends com.steamclock.steamclog.Redactable { *; }
-
-# Sample specific
 -keep public class * { # All public classes
     public static *; # All public static fields in those classes
     public protected abstract *(...); # All public or protected abstract methods in those classes
```

Comment:
I seem to still need this though to have the reflection used for `Redactatble` classes to work. @jacobminer if you know how I can combine these `keep` lines let me know, but for now this seems to be the only combination that works.